### ISSUE TITLE
TO_SEXP

### ISSUE THREAD

**Issue by FeggieBoss** [JetBrains-Research/Lama#26]
https://github.com/PLTools/Lama/blob/be0b32f7b9c75e61eff377cd34d4f65ebeeca204/runtime/runtime_common.h#L28
https://github.com/PLTools/Lama/blob/be0b32f7b9c75e61eff377cd34d4f65ebeeca204/runtime/runtime_common.h#L29
Is it suppose to be  SEXP_HEADER_SZ (```(sizeof(int) + sizeof(size_t) + sizeof(int))```) in TO_SEXP?

**Comment by danyaberezun** [comment]
No. It may be an unfortunate naming but there is no bug here. `TO_SEXP` accepts a raw pointer which in case of s-expression is actually a pointer to s-expression's tag rather than to "raw" data.
